data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES ('0', 'Agumon', '2020-02-03', '0', 'true', '10.23');
+INSERT INTO animals VALUES ('1', 'Gabumon', '2018-11-15', '2', 'true', '8');
+INSERT INTO animals VALUES ('2', 'Biyomon', '2021-01-07', '1', 'false', '15.04');
+INSERT INTO animals VALUES ('3', 'Tentomon', '2017-05-012', '5', 'true', '11');
```

Comment:
- [ ] It would be better if we input the right data as required
![Screenshot from 2023-07-19 23-01-23](https://github.com/MoyasiGinko/Vet-Clinic-Project/assets/46388113/374dc49e-f7ce-4b33-9a25-50834db68265)
![Screenshot from 2023-07-19 23-01-12](https://github.com/MoyasiGinko/Vet-Clinic-Project/assets/46388113/c491b992-cffb-4f71-842b-5a8be988d39c)